deb/publish.go

```diff
@@ -444,20 +654,25 @@ func (p *PublishedRepo) SourceNames() []string {
 	return sources
 }
 
-// UpdateLocalRepo updates content from local repo in component
-func (p *PublishedRepo) UpdateLocalRepo(component string) {
+// UpdateLocalRepo inserts/updates local repository source for component
+func (p *PublishedRepo) UpdateLocalRepo(component string, localRepo *LocalRepo) {
 	if p.SourceKind != SourceLocalRepo {
 		panic("not local repo publish")
 	}
 
-	item := p.sourceItems[component]
-	item.packageRefs = item.localRepo.RefList()
+	item, exists := p.sourceItems[component]
```

Comment:
what is the intention here ? why is the localRepo provided ? can there be multiple ?

Reply:
Because with this change, it is also possible to change the sources of published local repositories. That was not possible before the change.